src/mdtest.c

```diff
@@ -1871,44 +1880,10 @@ static void mdtest_iteration(int i, int j, MPI_Group testgroup, mdtest_results_t
 }
 
 void mdtest_init_args(){
-   barriers = 1;
-   branch_factor = 1;
-   throttle = 1;
-   stoneWallingStatusFile = NULL;
-   create_only = 0;
-   stat_only = 0;
-   read_only = 0;
-   verify_read = 0;
-   verification_error = 0;
-   remove_only = 0;
-   leaf_only = 0;
-   depth = 0;
-   num_dirs_in_tree = 0;
-   items_per_dir = 0;
-   random_seed = 0;
-   print_time = 0;
-   print_rate_and_time = 0;
-   shared_file = 0;
-   files_only = 0;
-   dirs_only = 0;
-   pre_delay = 0;
-   unique_dir_per_task = 0;
-   time_unique_dir_overhead = 0;
-   items = 0;
-   num_dirs_in_tree_calc = 0;
-   collective_creates = 0;
-   print_all_proc = 0;
-   write_bytes = 0;
-   stone_wall_timer_seconds = 0;
-   read_bytes = 0;
-   sync_file = 0;
-   call_sync = 0;
-   path_count = 0;
-   nstride = 0;
-   make_node = 0;
-#ifdef HAVE_LUSTRE_LUSTREAPI
-   global_dir_layout = 0;
-#endif /* HAVE_LUSTRE_LUSTREAPI */
+   memset(& o, 0, sizeof(o));
```

Comment:
Using a struct initializer will set all of the fields to zero except those specified:

    o = {
       .barriers = 1,
       .branch_factor = 1,
       .throttle = 1,
    };

Reply:
Very good suggestion, made.